scripts/superpmi.md

```diff
@@ -0,0 +1,64 @@
+### An overview of using superpmi.py
```

Comment:
The markdown is unfortunately github specific, so it will only render nicely on github. As for the support platforms, I will update the table when the default location is more concrete.